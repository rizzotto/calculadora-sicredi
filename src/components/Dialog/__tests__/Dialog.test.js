import React from "react";
import { Provider } from "react-redux";
import renderer from "react-test-renderer";

import store from "../../../providers/Store";
import Dialog from "../index.js";

it("renders Dialog correctly", () => {
    const tree = renderer
        .create(
            <Provider store={store}>
                <Dialog />
            </Provider>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
