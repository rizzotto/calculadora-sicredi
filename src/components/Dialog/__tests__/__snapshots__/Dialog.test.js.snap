// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Dialog correctly 1`] = `null`;
